app/views/journeys/expenses/financialCharges/FinancialChargesAmountView.scala.html

```diff
@@ -36,7 +36,7 @@
 
 @layout(pageTitle = title(form, messages(s"financialChargesAmount.title.$userType"))) {
 
-    @formHelper(action = FinancialChargesAmountController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
+    @formHelper(action = routes.FinancialChargesAmountController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
```

Comment:
importing the routes in this way stops errors when using IntelliJ Ultimate edition